pkgs/top-level/all-packages.nix

```diff
@@ -1917,6 +1917,8 @@ in
 
   fuzzel = callPackage ../applications/misc/fuzzel { };
 
+  flashfocus = python3Packages.callPackage ../misc/flashfocus { };
```

Comment:
Don't use `python3Packages.callPackage` in `all-packages.nix`!
Can someone please fix that?

Reply:
https://github.com/NixOS/nixpkgs/pull/117027